scripts/build.sh

```diff
@@ -19,4 +19,12 @@ export CGO_ENABLED=0
 
 # Build binary
 cd $GOPATH/src/github.com/raintank/metrictank
-go build -ldflags "-X main.GitHash=$GITVERSION" -o $BUILDDIR/metrictank
+
+OUTPUT=$BUILDDIR/metrictank
+
+if [ "$METRICTANK_BUILD" == "race" ]
```

Comment:
i'm personally not a big fan of setting environment variables to control the build, because:
* either i set the env vars once, and it affects all future builds in that terminal, often leading to confusion and ambiguity, especially with multiple terminals involved, having to recheck the values of the vars. it's basically the problem of state side-effects.

* or i run all commands with a `METRICTANK_BUILD=race` prefix, which is verbose.

So I think just running `make bin` or `make bin-race` explicitly is more explicit and simple.